Add documentation for the new theming mechanism (#6697)
surveyjs/survey-library: docs/get-started-vue.md

## Configure Styles

SurveyJS Form Library ships with several predefined themes illustrated below and a flexible theme customization mechanism based upon CSS variables.

![Themes in SurveyJS Form Library](images/survey-library-themes.png)

> To add SurveyJS themes to your application, open the Vue component in which your survey will be and import the Form Library style sheet:

Reviewer: ```suggestion
To add SurveyJS themes to your application, open the Vue component that will render your survey and import the Form Library style sheet:
```
